.goreleaser.yml

```diff
@@ -27,7 +29,9 @@ builds:
     main: ./cmd/hcloud/main.go
     binary: hcloud
     ldflags:
-      - -w -X github.com/hetznercloud/cli/internal/version.Version={{.Version}}
+      - -w
+      - -X {{ .ModulePath }}/internal/version.Version={{ .Version }}
```

Comment:
Not quite sure if this is needed. I think we can remove it since release-please sets the version variable anyway.

https://github.com/hetznercloud/packer-plugin-hcloud/blob/8a95e8f92abb326cddada38da8472bb7425e9b0e/.goreleaser.yml#L17

Reply:
True, but let's keep it so its more explicit.

Reply:
Could there be a conflict if release-please builds a new version but the tag is not yet there? Or does release-please always add the tag first?